sda-commons-server-auth-testing/src/test/java/org/sdase/commons/server/auth/testing/KeyLoaderProxyIT.java

```diff
@@ -25,16 +25,15 @@
 import org.junit.rules.RuleChain;
 import org.sdase.commons.server.auth.testing.test.AuthTestApp;
 import org.sdase.commons.server.auth.testing.test.AuthTestConfig;
-import org.sdase.commons.server.testing.EnvironmentRule;
 import org.sdase.commons.server.testing.SystemPropertyRule;
 
 /** A test that checks if the jersey client that is used to load keys can use a proxy server */
 public class KeyLoaderProxyIT {
   private static final WireMockClassRule PROXY_WIRE =
       new WireMockClassRule(wireMockConfig().dynamicPort());
 
-  private static final EnvironmentRule ENVIRONMENT_RULE =
-      new EnvironmentRule().setEnv("AUTH_RULE", "{\"keys\": [{}]}");
+  private static final SystemPropertyRule SYSTEM_PROPERTY_RULE =
```

Comment:
We have two system property rules now. Can we join them into one?

Reply:
Yes, done.